.github/workflows/paths-ignore.yaml

```diff
@@ -41,18 +44,16 @@ jobs:
     - name: Determine paths to ignore
       if: github.event_name == 'pull_request'
       run: |
-        curl -L -o yq \
-          https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
-        chmod a+x yq
         PATHS_IGNORE="PATHS_IGNORE="
         while read -r line; do
-          PATHS_IGNORE="${PATHS_IGNORE} ':!/${line}'"
-        done <<< "$(./yq '.[]' <<< "$INPUT")"
+          if [[ -n $line ]]; then
+            PATHS_IGNORE="${PATHS_IGNORE} :!/${line}"
```

Comment:
I'm wondering if this going to work with glob patterns including whitespace, but I guess they will just have to be escaped:

```
paths-ignore-globs: |
  My\\ Temp\\ Dir/**
```

Not really worried because I don't see us adding directory names with whitespace to the repo anyways. 😄